src/main/scala/beam/utils/scenario/urbansim/censusblock/merger/PlanMerger.scala

```diff
@@ -9,7 +9,7 @@ class PlanMerger(modeMap: Map[String, String]) extends Merger[InputPlanElement,
 
   private def transform(inputPlanElement: InputPlanElement): PlanElement = {
     PlanElement(
-      inputPlanElement.tripId,
+      Option(inputPlanElement.tripId).get.toString,
```

Comment:
What is the purpose of transforming a value into an Option and then immediately getting the value out of option?
`Option(value).get` is equal to `value` but require CPU time

Reply:
make sense, will adjust this to be value.